README.md

```diff
@@ -34,29 +36,23 @@
 
 [Nest](https://github.com/nestjs/nest) framework TypeScript starter repository.
 
-## Installation
+## Installation with docker compose
 
 ```bash
-$ npm install
+$ docker-compose build
 ```
-
-## Running the app
+## Running the app with docker
 
 ```bash
-# development
-$ npm run start
-
-# watch mode
-$ npm run start:dev
-
-# production mode
-$ npm run start:prod
+$ docker-compose up
 ```
 
 ## Usage
 
 ```bash
 The issue creator sits at localhost:3500. 
-Send your logs to localhost:3500
+If installed with docker the database is available at localhost:27017.
```

Comment:
was genau sollte ich unter dieser url vorfinden? ich erhalte im moment nur die meldung `It looks like you are trying to access MongoDB over HTTP on the native driver port.` nicht ganz das, was ich erwartet habe.

Reply:
Der Hinweis ist nur dafür gedacht, falls man über ein GUI Tool auf die Datenbank zugreifen möchte. Das könnte ich aber auch noch expliziter ausdrücken.